helm/cluster-eks/values.schema.json

```diff
@@ -169,12 +169,6 @@
                             "configTemplateName": "EKSCorednsHelmValues",
                             "enable": true
                         },
-                        "etcdDefrag": {
-                            "enable": true
-                        },
-                        "etcdKubernetesResourcesCountExporter": {
-                            "enable": true
-                        },
```

Comment:
all apps default to `enable: false` in the cluster chart so we just remove the override